nautobot/extras/tests/test_customfields.py

```diff
@@ -475,17 +476,22 @@ def setUpTestData(cls):
         if "example_plugin" in settings.PLUGINS:
             cls.cf_plugin_field = CustomField.objects.get(name="example_plugin_auto_custom_field")
 
-        statuses = Status.objects.get_for_model(Site)
+        statuses = Status.objects.get_for_model(Location)
 
-        # Create some sites
-        cls.sites = (
-            Site.objects.create(name="Site 1", slug="site-1", status=statuses.get(slug="active")),
-            Site.objects.create(name="Site 2", slug="site-2", status=statuses.get(slug="active")),
+        # Create some locations
+        cls.lt = LocationType.objects.get(name="Campus")
```

Comment:
How badly will it break things if we update the location/location-type factories to create region-level and site-level locations? Obviously not part of this PR but something to think about...